### PR TITLE
Bump to Ember 2.13.3

### PR DESCRIPTION
**Pull request by mixonic**
Mostly I've taken my lead from https://github.com/tastejs/todomvc/pull/1722

/cc @cibernox

### CODE REVIEW

File: examples/emberjs/todomvc/README.md [hunk 1]
```diff
@@ -7,9 +7,12 @@
...
 ## Note for people updating this app.
 
-The `index.html` and the `assets` folder in the parent folder as simlinks into the items with the
-same names inside `dist`. The `dist` folder has to be checked in git and built for production.
+The `index.html` and the `assets` folder in the parent folder as simlinks into
+the items with the same names inside `dist`. The `dist` folder has to be checked
+in git and built for production.
 
-To work on this comment `<base href="/examples/ember-cli/index.html" />` in the `app/index.html`
-and uncommented it back before doing the production build.
```
Comment (mixonic) [inline examples/emberjs/todomvc/README.md]: I've removed this documentation, as I believe it is out of date.

File: examples/emberjs/todomvc/config/targets.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
+    'ie 11',
```
Comment (mixonic) [inline examples/emberjs/todomvc/config/targets.js]: Ember can be built back to `'ie 9'` here. I chose IE11 somewhat arbitrarily.

File: examples/emberjs/todomvc/config/environment.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
-/* jshint node: true */
+/* eslint-env node */
 
 module.exports = function (environment) {
 	var ENV = {
 		modulePrefix: 'todomvc',
 		environment: environment,
-		baseURL: null,
```
Comment (mixonic) [inline examples/emberjs/todomvc/config/environment.js]: Hm, did I miscalculate and this must be `null` to make the `index.html` file have no leading `/`?

Comment (mixonic) [inline examples/emberjs/todomvc/config/environment.js]: `index.html` paths look correct with `rootURL`. I believe `baseURL` is deprecated 🤔

File: examples/emberjs/todomvc/.travis.yml [hunk 4]
```diff
@@ -1,25 +1,21 @@
 ---
```
Comment (FadySamirSadek) [inline examples/emberjs/todomvc/.travis.yml]: Why do we need a .travis.yml file inside this directory ?

File: examples/emberjs/todomvc/README.md (hunk 1 above)
Comment (FadySamirSadek) [reply examples/emberjs/todomvc/README.md]: I think this README.md file should be moved to the root directory of the example

File: examples/emberjs/todomvc/config/environment.js (hunk 3 above)
Comment (FadySamirSadek) [reply examples/emberjs/todomvc/config/environment.js]: Yes, I think that baseURL is deprecated. The important thing is that it works 🙂

File: examples/emberjs/todomvc/package.json [hunk 5]
```diff
@@ -16,28 +16,28 @@
...
+    "ember-export-application-global": "^2.0.0",
+    "ember-inflector": "2.0.1",
+    "ember-load-initializers": "^1.0.0",
+    "ember-resolver": "^4.0.0",
+    "ember-source": "~2.13.3",
+    "loader.js": "^4.2.3"
   },
   "engines": {
-    "node": ">= 0.12.0"
+    "node": ">= 6"
```
Comment (FadySamirSadek) [inline examples/emberjs/todomvc/package.json]: I think we should lock this version WDYT?

**Comment by zoltan-nz** [comment]
How about this PR? https://github.com/tastejs/todomvc/pull/1912

File: examples/emberjs/todomvc/README.md (hunk 1 above)
Comment (zoltan-nz) [reply examples/emberjs/todomvc/README.md]: Ah, I had the same feeling... I update the readme as Mixonic suggested and I move it to the root directory.

**Comment by mixonic** [comment]
Closing in favor of https://github.com/tastejs/todomvc/pull/1912

**Pull request closed without merging**